src/services/api.services.js

```diff
@@ -0,0 +1,104 @@
+import { API_BASE_URL } from "../constants/url.config";
+const token = window.localStorage.getItem("token");

```

Comment:
Lets create a separate service(e.g tokenService) for handling all token related operations.